.github/actions/full-release/action.yml

```diff
@@ -0,0 +1,39 @@
+name: Build, Test, and Publish
```

Comment:
This was copied from [here](https://github.com/launchdarkly/js-core/blob/main/actions/full-release/action.yml). Is all this required to publish to npm?